src/AmbientSounds/Attributes/IsExternalInit.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Reserved to be used by the compiler for tracking metadata.
+    /// This class should not be used by developers in source code.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    internal static class IsExternalInit
```

Comment:
what's this for?

Reply:
That's just an internal type that's needed by Roslyn to enable the records feature (specifically, init-only properties). We need to define that type ourselves on UWP since it doesn't come out of the box here (it's only included on .NET 5). Put simply, we only need that type to be there to make the C# compiler happy, but you can just ignore it 🙂